app/src/main/java/org/sopt/dosopttemplate/HomeFragment.kt

```diff
@@ -42,6 +43,17 @@ class HomeFragment : Fragment() {
         val concatAdapter = ConcatAdapter(userAdapter, friendAdapter)
         //concat된 list를 fragment영역에 적용
         binding.rvFriends.adapter = concatAdapter
+
+        //fab 상단 스크롤 지원
+        upScrollListener(binding.rvFriends)
+    }
+
+    fun upScrollListener(view: RecyclerView?) {
+        //fab 클릭시 스크롤 상단으로 이동
+        binding.fabToTop.setOnClickListener {
+            binding.rvFriends.smoothScrollToPosition(0)
+        }
+
     }
```

Comment:
```suggestion
    fun upScrollListener(view: RecyclerView?) =with(binding) {
        //fab 클릭시 스크롤 상단으로 이동
        fabToTop.setOnClickListener {
            rvFriends.smoothScrollToPosition(0)
        }

    }
```
이렇게도 쓸 수 있답니다(찬우오빠가 알려준 꿀팁)

Reply:
어? 와!!!!! 
함 차차 반영해보도록 하겠습니다👍👍